test/data/error_file.csv

```diff
@@ -1,6 +1,15 @@
+just some, junky pre-header nonsense
+
+empty lines were added for sample error CSV position tests
+
```

Comment:
nevermind the other comment this makes sense

Reply:
oh